packages/dcos-checks/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e  # Fail the script if anything fails
+set -x  # Verbose output
+set -u  # Undefined variables
+
+CHECKS_PATH=$GOPATH/src/github.com/dcos/dcos/packages/dcos-checks/extra/checks
```

Comment:
Can we call the go package `dcos-checks` so it matches up with the pkgpanda package?